### PR TITLE
add unit test for function in pkg/labels/labels.go

### PR DESCRIPTION
**Pull request by Hellcatlk**
Signed-off-by: ZouYu <zouy.fnst@cn.fujitsu.com>

### CODE REVIEW

File: pkg/labels/labels_test.go [hunk 1]
```diff
@@ -19,6 +19,144 @@ import (
...
+		t.Run(c.name, func(t *testing.T) {
+			str := c.lables.String()
+			if str != c.expected {
+				t.Errorf("expected: %v, got: %v", c.expected, str)
+			}
+		})
+	}
+}
+
+func TestLabels_MarshalJSON(t *testing.T) {
```
Comment (codesome) [inline pkg/labels/labels_test.go:63]: The tests for marshal and unmarshal of json would belong to the JSON library, it is not needed here.

Comment (Hellcatlk) [reply pkg/labels/labels_test.go:63]: Thanks, i will delete it.

File: pkg/labels/labels_test.go [hunk 2]
```diff
@@ -19,6 +19,47 @@ import (
...
+			lables:   nil,
+			expected: "{}",
+		},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			str := c.lables.String()
+			if str != c.expected {
+				t.Errorf("expected: %v, got: %v", c.expected, str)
+			}
```
Comment (codesome) [inline pkg/labels/labels_test.go:58]: Can you use `testutil.Equal` instead?

Comment (Hellcatlk) [reply pkg/labels/labels_test.go]: Done

File: pkg/labels/labels_test.go [hunk 3]
```diff
@@ -19,6 +19,39 @@ import (
...
+			expected: "{}",
+		},
+		{
+			lables:   nil,
+			expected: "{}",
+		},
+	}
+	for _, c := range cases {
+		str := c.lables.String()
+		testutil.Equals(t, c.expected, str, "expected: %v, got: %v", c.expected, str)
```
Comment (codesome) [inline pkg/labels/labels_test.go:51]: Printing is done in that method :)
```suggestion
		testutil.Equals(t, c.expected, str)
```

Comment (Hellcatlk) [reply pkg/labels/labels_test.go]: Done

**Comment by codesome** [comment]
Thanks!

**Pull request merged**